android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -324,6 +324,10 @@ public void onDownloadStart(String url, String userAgent, String contentDisposit
             fileName = URLUtil.guessFileName(url, contentDisposition, null);
           }
 
+          if (fileName == "") {
```

Comment:
자바라서 아래처럼 처리하도록 수정해야 할 거 같습니다

```suggestion
          if (fileName.equals("")) {
```

Reply:
제안해주신 방법으로 수정했습니다. 감사합니다.